### ISSUE TITLE
[tool_crash] RPCError

### ISSUE THREAD

**Issue by zygaPG** [flutter/flutter#156255]
## Command
```sh
flutter run
```

## Steps to Reproduce
1. ...
2. ...
3. ...

## Logs
RPCError
```console
#0      VmService._call (package:vm_service/src/vm_service.dart:1774:7)
#1      VmService.callServiceExtension (package:vm_service/src/vm_service.dart:1726:14)
#2      VmService.callMethod (package:vm_service/src/vm_service.dart:1717:12)
#3      FlutterVmService.callMethodWrapper (package:flutter_tools/src/vmservice.dart:501:28)
#4      FlutterVmService.getFlutterViews (package:flutter_tools/src/vmservice.dart:913:51)
#5      HotRunner.attach (package:flutter_tools/src/run_hot.dart:304:63)
<asynchronous suspension>
#6      RunCommand.runCommand (package:flutter_tools/src/commands/run.dart:867:27)
<asynchronous suspension>
#7      FlutterCommand.run.<anonymous closure> (package:flutter_tools/src/runner/flutter_command.dart:1408:27)
<asynchronous suspension>
#8      AppContext.run.<anonymous closure> (package:flutter_tools/src/base/context.dart:153:19)
<asynchronous suspension>
#9      CommandRunner.runCommand (package:args/command_runner.dart:212:13)
<asynchronous suspension>
#10     FlutterCommandRunner.runCommand.<anonymous closure> (package:flutter_tools/src/runner/flutter_command_runner.dart:420:9)
<asynchronous suspension>
#11     AppContext.run.<anonymous closure> (package:flutter_tools/src/base/context.dart:153:19)
<asynchronous suspension>
#12     FlutterCommandRunner.runCommand (package:flutter_tools/src/runner/flutter_command_runner.dart:364:5)
<asynchronous suspension>
#13     run.<anonymous closure>.<anonymous closure> (package:flutter_tools/runner.dart:130:9)
<asynchronous suspension>
#14     AppContext.run.<anonymous closure> (package:flutter_tools/src/base/context.dart:153:19)
<asynchronous suspension>
```
```console
[32m[✓][39m Flutter (Channel stable, 3.24.0, on Microsoft Windows [Version 10.0.22631.4169], locale pl-PL)
    [32m•[39m Flutter version 3.24.0 on channel stable at C:\Users\zygaPG\Desktop\Flutter SDK\flutter
    [32m•[39m Upstream repository https://github.com/flutter/flutter.git
    [32m•[39m Framework revision 80c2e84975 (9 weeks ago), 2024-07-30 23:06:49 +0700
    [32m•[39m Engine revision b8800d88be
    [32m•[39m Dart version 3.5.0
    [32m•[39m DevTools version 2.37.2

[32m[✓][39m Windows Version (Installed version of Windows is version 10 or higher)

[33m[!][39m Android toolchain - develop for Android devices (Android SDK version 35.0.0)
    [32m•[39m Android SDK at C:\Users\zygaPG\AppData\Local\Android\sdk
    [31m✗[39m cmdline-tools component is missing
      Run `path/to/sdkmanager --install "cmdline-tools;latest"`
      See https://developer.android.com/studio/command-line for more details.
    [31m✗[39m Android license status unknown.
      Run `flutter doctor --android-licenses` to accept the SDK licenses.
      See https://flutter.dev/to/windows-android-setup for more details.

[32m[✓][39m Chrome - develop for the web
    [32m•[39m Chrome at C:\Program Files\Google\Chrome\Application\chrome.exe

[32m[✓][39m Visual Studio - develop Windows apps (Visual Studio Build Tools 2022 17.9.6)
    [32m•[39m Visual Studio at C:\Program Files (x86)\Microsoft Visual Studio\2022\BuildTools
    [32m•[39m Visual Studio Build Tools 2022 version 17.9.34728.123
    [32m•[39m Windows 10 SDK version 10.0.22621.0

[32m[✓][39m Android Studio (version 2024.1)
    [32m•[39m Android Studio at C:\Program Files\Android\Android Studio
    [32m•[39m Flutter plugin can be installed from:
      🔨 https://plugins.jetbrains.com/plugin/9212-flutter
    [32m•[39m Dart plugin can be installed from:
      🔨 https://plugins.jetbrains.com/plugin/6351-dart
    [32m•[39m Java version OpenJDK Runtime Environment (build 17.0.11+0--11852314)

[32m[✓][39m VS Code (version 1.94.0)
    [32m•[39m VS Code at C:\Users\zygaPG\AppData\Local\Programs\Microsoft VS Code
    [32m•[39m Flutter extension version 3.98.0

[32m[✓][39m Connected device (4 available)
    [32m•[39m sdk gphone64 x86 64 (mobile) • emulator-5554 • android-x64    • Android 15 (API 35) (emulator)
    [32m•[39m Windows (desktop)            • windows       • windows-x64    • Microsoft Windows [Version
      10.0.22631.4169]
    [32m•[39m Chrome (web)                 • chrome        • web-javascript • Google Chrome 128.0.6613.138
    [32m•[39m Edge (web)                   • edge          • web-javascript • Microsoft Edge 129.0.2792.65

[32m[✓][39m Network resources
    [32m•[39m All expected network resources are available.

[33m![39m Doctor found issues in 1 category.

```

## Flutter Application Metadata
**Type**: app
**Version**: 1.0.0+1
**Material**: true
**Android X**: false
**Module**: false
**Plugin**: false
**Android package**: null
**iOS bundle identifier**: null
**Creation channel**: stable
**Creation framework version**: 300451adae589accbece3490f4396f10bdf15e6e
### Plugins
path_provider-2.1.4
path_provider_android-2.2.10
path_provider_foundation-2.4.0
path_provider_linux-2.2.1
path_provider_windows-2.3.0
shared_preferences-2.3.2
shared_preferences_android-2.3.2
shared_preferences_foundation-2.5.2
shared_preferences_linux-2.4.1
shared_preferences_web-2.4.2
shared_preferences_windows-2.4.1
uni_links-0.5.1
uni_links_web-0.1.0

**Comment by TheCodeDaniel** [comment]
Hello @zygaPG 

Can you please drop a code snippet, most especially the code that is being called as app begins it's run time ?

**Comment by danagbemava-nc** [comment]
Hi @zygaPG, this should have been fixed in 3.24.2, can you upgrade to 3.24.3 to see if you still experience this issue?
